tests/test_port_lt.py

```diff
@@ -0,0 +1,139 @@
+import time
+import os
+import pytest
+
+from swsscommon import swsscommon
+
+
+class TestPortLinkTraining(object):
+    def test_PortLinkTrainingForce(self, dvs, testlog):
+
+        db = swsscommon.DBConnector(0, dvs.redis_sock, 0)
+        adb = dvs.get_asic_db()
+
+        tbl = swsscommon.ProducerStateTable(db, "PORT_TABLE")
+        fvs = swsscommon.FieldValuePairs([("link_training","off")])
+        tbl.set("Ethernet0", fvs)
+
+        tbl = swsscommon.ProducerStateTable(db, "PORT_TABLE")
+        fvs = swsscommon.FieldValuePairs([("link_training","on")])
+        tbl.set("Ethernet4", fvs)
+
+        # validate if autoneg false is pushed to asic db when set first time
```

Comment:
Typo in comments, should be link-training instead of autoneg

Reply:
Thanks, will fix it